src/Plots.jl

```diff
@@ -106,6 +106,15 @@ export
 
 # ---------------------------------------------------------
 
+import NaNMath
+# define functions (e.g. `_extrema`, that ignores NaNs, when the type is applicable. To overcome the destructive effects of https://github.com/JuliaLang/julia/pull/12563
+for fun in (:extrema, :minimum, :maximum)
```

Comment:
An alternative to the eval magic would be to just write it out - less concise, but probably easier for people wondering where `_minimum` is defined and what it does. Thoughts?